gradle/verification-metadata.xml

```diff
@@ -1473,20 +1383,12 @@
          <artifact name="tini-0.19.0-amd64.jar">
             <sha256 value="93dcc18adc78c65a028a84799ecf8ad40c936fdfc5f2a57b1acda5a8117fa82c" origin="Generated by Gradle"/>
          </artifact>
-         <artifact name="tini-0.19.0-arm64.jar">
```

Comment:
It's very interesting to me that we had two different `sha256` values here.

Reply:
This is intentional. One is the x86 artifact and the other is for arm. These are used on our docker images and both are needed.

Reply:
OMG, the number of times I read that as `amd64` *twice*. Yarp, they're different. 😵‍💫

Reply:
Added back via 09c5e430fc0e769607509b6ef5a441bfe3ee0a0c